code/mathematical-algorithms/factorial/factorial.py

```diff
@@ -0,0 +1,10 @@
+//factorial without using recursion
```

Comment:
Please add comment  /* Part of Cosmos by OpenGenus Foundation */ at the top of every code you contribute to this project  ! 👍

Reply:
@yatharthshahjpr The comment needs to start with a `#`